src/js/services/bootstrap.js

```diff
@@ -72,7 +73,10 @@ export default App.extend({
     return this.organization;
   },
   getDirectories() {
-    return this.directories;
+    return this.directories.clone();
```

Comment:
not really part of the PR, but any collection from bootstrap should really be cloned..